frodo.js

```diff
@@ -1,76 +1,82 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!